eregs_core/management/commands/cache_content.py

```diff
@@ -0,0 +1,59 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ObjectDoesNotExist
+from eregs_core.models import *
+from eregs_core.utils import xml_to_json
+from lxml import etree
+from itertools import product, chain
+
+import os
+import json
+import requests
+
+
+class Command(BaseCommand):
+
+    help = 'Cache the views of the specified regulation.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('regs', nargs='*', type=str)
+
+    def handle(self, *args, **options):
+        regs_to_cache = options['regs']
+        preambles = Preamble.objects.filter(tag='preamble').exclude(reg_version__version=None)
+        base = 'http://localhost:8000'
```

Comment:
RequestFactory can be used to create a mock request to pass into view code, if, say, you wanted to invoke your views programmatically. If you want to just hit the URLs an easier pattern is to use [the Django test client](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#the-test-client):

```py
from django.test import Client

client = Client()
for url in urls_to_warm:
   response = client.get(url)
```

This invokes the view code against whatever local Django instance is running this code (database/cache).